app/assets/javascripts/controllers/middleware_deployment/middleware_deployment_controller.js

```diff
@@ -20,16 +20,23 @@ function MwAddDeploymentController($scope, $http, miqService) {
     })
       .then(
         function(result) { // success
+          if (result.data.error_type === 'EXISTS') {
```

Comment:
perhaps there is a way to use constants for things like error_type to reduce the chances of typos?